www/xyerv00/projekt_1/charlotte.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>V&V kabáty - kabát Charlotte</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
+    <link rel="icon" href="favicon.ico" type="image/x-icon">
+    <link type="text/css" media="screen" rel="stylesheet" href="css/style-product-item.css">
```

Comment:
nemelo by to byt tak, ze kazda ma vlastni css soubor, v ramci jednoho webu je lepsi mit jen jeden stylesheet, ktery bude platit pro vsechny stranky, urcite se v nich neco opakuje, proto jen jeden css soubor